examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_runner.hpp

```diff
@@ -331,15 +332,16 @@ template <class GemmKernel> struct ExampleRunner {
     size_t workspace_size = GemmKernel::get_workspace_size(arguments);
     cutlass::device_memory::allocation<uint8_t> workspace(workspace_size);
 
-    GemmKernel::can_implement(arguments);
+    if (!GemmKernel::can_implement(arguments)) {
+      std::cout << "Invalid Problem Size: " << options.batch << 'x' << options.num_heads << 'x' <<
+        options.seq_len << 'x' << options.head_size << (options.is_causal ? "xCausal" : "xNonCausal") << std::endl;
+      std::exit(1);
+    }
```

Comment:
```suggestion
    if (!GemmKernel::can_implement(arguments)) {
      std::cout << "Invalid Problem Size: " << options.batch << 'x' << options.num_heads << 'x' <<
        options.seq_len << 'x' << options.head_size << (options.is_causal ? "xCausal" : "xNonCausal") << std::endl;
      return Status::kErrorInvalidProblem;
    }
```
or maybe `kErrorInvalidLayout`? or just replace with `CUTLASS_CHECK` (which seems simplest).